chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -120,6 +120,14 @@
         <partitions config:type="list">
           <partition>
             <create config:type="boolean">true</create>
+            <format config:type="boolean">true</format>
+           <filesystem config:type="symbol">vfat</filesystem>
+           <partition_id config:type="integer">259</partition_id>
+            <mount>/boot/efi</mount>
+            <size>128M</size>
+          </partition>
+          <partition>
+            <create config:type="boolean">true</create>
```

Comment:
ah true, we used this trick here: https://github.com/crowbar/crowbar-core/blob/master/chef/cookbooks/crowbar-hacks/recipes/default.rb#L81